Mathlib/RingTheory/Algebraic.lean

```diff
@@ -136,23 +136,23 @@ theorem isAlgebraic_of_mem_rootSet {R : Type u} {A : Type v} [Field R] [Field A]
 
 open IsScalarTower
 
-theorem isAlgebraic_algebraMap_of_isAlgebraic {a : S} :
+protected theorem IsAlgebraic.algebraMap {a : S} :
```

Comment:
I assume the `protected` was not handled by the tool?

Reply:
Correct, although there is the possibility of flagging declarations that you want to protect in a future version of the renaming tool: the `ilean` files that I use contains where the declaration is defined.

Reply:
However, more basic features are missing in the tool, so I haven't given much thought to this.